src/handlers.jl

```diff
@@ -84,11 +110,7 @@ function complete_request(socket, msg)
         for c in comps
             try
                 t = eval(Main, parse("typeof($c)"))
-                if issubtype(t, Function)
-                    push!(typeMap, Dict("text" => c, "type" => "function"))
-                else
-                    push!(typeMap, Dict("text" => c, "type" => "$t"))
-                end
+                push!(typeMap, Dict("text" => c, "type" => "$(complete_type(t))"))
```

Comment:
No need for interpolation since `complete_type` already returns a string.  Just use `"type" => complete_type(t)`